statistics/table.go

```diff
@@ -286,9 +286,9 @@ func getPseudoRowCountByIndexRanges(sc *variable.StatementContext, indexRanges [
 		totalCount += count
 	}
 	// To avoid the totalCount become too small.
-	if uint64(totalCount) < 1000 {
+	if totalCount < tableRowCount/1000 {
```

Comment:
Since the `@@tidb_max_row_count_for_inlj` is 128, and in test we only build the statistic for pk column. If don't change this, test won't pass.
Comment updated.